libraries/AP_NavEKF3/AP_NavEKF3_core.cpp

```diff
@@ -1417,7 +1417,7 @@ void NavEKF3_core::ConstrainVariances()
     }
 
     if (!inhibitDelVelBiasStates) {
-        for (uint8_t i=13; i<=15; i++) P[i][i] = constrain_float(P[i][i],0.0f,sq(10.0f * dtEkfAvg));
+        for (uint8_t i=13; i<=15; i++) P[i][i] = constrain_float(P[i][i],1e-9f,sq(10.0f * dtEkfAvg));
```

Comment:
No, the 1e-9 was determined from testing and is related to overall stability of the covariance matrix prediction and updates.